src/Sulu/Bundle/ContactBundle/Entity/Account.php

```diff
@@ -145,7 +145,7 @@ class Account implements AccountInterface
     protected $parent;
 
     /**
-     * @var string
+     * @var Address[]
```

Comment:
By phpstan this should be somthing like:

```suggestion
     * @var Collection<int, Address>
```